getting-started/configuration.md

```
@@ -363,46 +363,52 @@ This configuration can be set when the module loads or at runtime.
 
 `VKEY_MAX_ENTITY_COUNT` is 100,000.
 
+#### Example
+
+```sh
+$ redis-server --loadmodule ./falkordb.so VKEY_MAX_ENTITY_COUNT 50000
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)